src/components/RadioButton/RadioButton.vue

```diff
@@ -0,0 +1,55 @@
+<template lang="pug">
```

Comment:
I think that radio button should be a button first, not sure we can use bootstrap button. We should discuss about this one